doc/RELEASE-0.8.txt

```diff
@@ -0,0 +1,20 @@
+============
+Release v0.8
+============
+
+We are pleased to announce the release of screed v0.8. screed is a database
```

Comment:
On Sat, Mar 14, 2015 at 07:58:27AM -0700, bocajnotnef wrote:
> > @@ -0,0 +1,20 @@
> > +============
> > +Release v0.8
> > +============
> > +
> > +We are pleased to announce the release of screed v0.8. screed is a database
> 
> Alright. It wasn't capitalized in the past release notes for screed.

Indeed, it wasn't :)